base/logging.jl

```diff
@@ -490,7 +490,7 @@ function handle_message(logger::SimpleLogger, level, message, _module, group, id
     end
     buf = IOBuffer()
     iob = IOContext(buf, logger.stream)
-    levelstr = string(level)
+    levelstr = level == Warn ? "Warning" : string(level)
```

Comment:
My thought to make this less exceptional was that one might be able to specify a `levelstr` for each kind of message instead of just hacking this in there. Not important though since extension points can always be added in the future.